x-pack/plugins/security_solution/public/management/pages/policy/view/ingest_manager_integration/endpoint_package_custom_extension/components/fleet_integration_event_filters_card.tsx

```diff
@@ -86,11 +86,7 @@ export const FleetIntegrationEventFiltersCard = memo<{
     isMounted.current = true;
     const fetchStats = async () => {
       try {
-        const summary = await eventFiltersApi.getList({
-          perPage: 1,
-          page: 1,
-          filter: parsePoliciesToKQL([policyId, 'all']),
-        });
+        const summary = await eventFiltersApi.getSummary(parsePoliciesToKQL([policyId, 'all']));
         if (isMounted.current) {
           setStats({
             total: summary.total,
```

Comment:
you could do:
`setStats(summary);`

Reply:
😄  yeah!

Reply:
done a08472e9178638a816431e1dd07fccfc883035f1